index.html

```diff
@@ -1901,6 +1901,51 @@ <h3>Securing Data Losslessly</h3>
           </p>
         </section>
 
+        <section>
+          <h3 id="cryptosuiteString">The `cryptosuiteString` Datatype</h3>
+
+          <p>
+This specification encodes cryptographic identifiers as enumerable strings,
+which is useful in processes, such as compression algorithms, that need to
+efficiently encode such strings. In environments that support data types for
+string values, such as RDF [[?RDF-CONCEPTS]], cryptographic identifier content
+is indicated using a literal value whose datatype is set to
+`https://w3id.org/security#cryptosuiteString`.
+          </p>
+
+          <p>
+The <code>cryptosuiteString</code> datatype is defined as follows:
+          </p>
+
+          <dl>
+            <dt>The URL denoting this datatype</dt>
+            <dd>
+is `https://w3id.org/security#cryptosuiteString`.
+            </dd>
+            <dt>The lexical space</dt>
+            <dd>
+is the set of all cryptosuite types expressed using American Standard Code
+for Information Interchange [[ASCII]] strings that are defined by the sum of all
```

Comment:
I wonder whether term "union" is not better than "sum" ("union" in the sense of a Union datatype in programming languages).

Reply:
Yes, "union" is clearly better, applied in a commit below.